core/controllers/ingest.py

```diff
@@ -98,7 +96,7 @@ def ingest(body: dict, excel_file: FileStorage) -> tuple[dict, int]:
     try:
         initial_validate(workbook_data, ingest_dependencies.initial_validation_schema, auth)
         if fund == "Towns Fund":
-            # TODO https://dluhcdigital.atlassian.net/browse/SMD-660: use tables to extract, process and validate TF
+            assert isinstance(ingest_dependencies, TFIngestDependencies)
```

Comment:
I'm never keen on assertions in production code flows particularly. If we think we need this check, can we swap this for a standard conditional and raise a ValueError or similar instead?

Reply:
Fair enough, have just made that change now